chapters/sampling-randomization-power.tex

```diff
@@ -1,386 +1,428 @@
+is guaranteed to be \textit{nonzero}:
+this is called the \textbf{in-sample} or \textbf{finite-sample correlation}.
+
+Since we know that the true correlation
+(over the ``population'' of potential samples or randomizations)
+is zero, we think of the observed correlation as an \textbf{error}.
+In sampling, we call this the \textbf{sampling error},
+and it is defined as the difference between the true population parameter
+and the observed mean due to chance selection of units.\sidenote{
+  \url{https://economistjourney.blogspot.com/2018/06/what-is-sampling-noise.html}}
+In randomization, we call this the \textbf{randomization noise},
+and define it as the difference between the true treatment effect
+and the estimated effect due to placing units in groups.
+The intuition for both measures is that from any group,
+you can find some possible subsets that have higher-than-average values of some measure;
+similarly, you can find some that have lower-than-average values.
+Your sample or randomization will inevitably fall in one of these categories,
+and we need to assess the likelihood and magnitude of this occurence.\sidenote{
+  \url{https://davegiles.blogspot.com/2019/04/what-is-permutation-test.html}}
+Power calculation and randomization inference are the two key tools to doing so.
+
+\subsection{Power calculations}
+
+Power calculations report the likelihood that your experimental design
+will be able to detect the treatment effects you are interested in.
+This measure of \textbf{power} can be described in various different ways,
+each of which has different practical uses.\sidenote{
+  \url{http://www.stat.columbia.edu/~gelman/stuff_for_blog/chap20.pdf}}
+The purpose of power calculations is to identify where the strengths and weaknesses
+of your design are located, so you know the relative tradeoffs you will face
+by changing your randomization scheme for the final design.
+They also allow realistic interpretations of evidence:
+results low-power studies can be very interesting,
+but they have a correspondingly higher likelihood
+of reporting false positive results.
+
+The classic definition of power
+is the likelihood that a design detects a significant treatment effect,
+given that there is a non-zero true effect in reality.
+This definition is useful retrospectively,
+but it can also be re-interpreted to help in experimental design.
+There are two common and useful practical applications
+of that definition that give actionable, quantitative results.
+The \textbf{minimum detectable effect (MDE)}\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Minimum_Detectable_Effect}}
+is the smallest true effect that a given study design can detect.
+This is useful as a check on whether a study is worthwhile.
+If, in your field, a ``large'' effect is just a few percentage points
+or a fraction of a standard deviation,
+then it is nonsensical to run a study whose MDE is much larger than that.
```

Comment:
Add more concrete example here, something like

> This is because, given the sample size and variation in the population, (or whatever other factors are at play here that I'm forgetting to list) any effect that can possibly be statistically detected needs to be much larger. So the study will not be able to say anything about the size interval that is economically relevant.

Ok, this is not good writing, but I'm just trying to think of a more intuitive way to put it, as my impression is that I only understood the concept as it was described in abstract terms here because I was already familiar with it.

Reply:
I thought that's not too bad: b4bb5bd1a0bea30ff852d867429eea33c9e46736